yt/odin/bin/yt_odin_webservice/__main__.py

```diff
@@ -151,6 +166,31 @@ def solomon_duration():
     return jsonify({"sensors": duration_sensors + timed_out_sensors})
 
 
+@app.route("/prometheus")
+def prometheus():
+    now = datetime.datetime.now(datetime.timezone.utc)
+    period = datetime.timedelta(seconds=int(request.args.get("period", 90)))
```

Comment:
If we are implementing this ourselves, can we put some link to docs in the comments? I actually do not know which HTTP request args prometheus handlers are supposed to support. For example, is there a parameter for "now"?

Reply:
It's not that hard to google these docs, but okay, mijn vriend, I'll do that!)
Prometheus doesn't use any params itself, but you can specify `metrics_path` and `params` in the scaper config, so it can look like:
```
metrics_path: "/prometheus"
    params:
      period: ["120"]
```

Reply:
Ok! While I am being picky, let's also add a comment about what `period` means logically, since it is basically an external configuration option. One should not have to read the code to understand what value should be provided for it.